monai/utils/module.py

```diff
@@ -564,11 +567,11 @@ def version_leq(lhs: str, rhs: str) -> bool:
     """
 
     lhs, rhs = str(lhs), str(rhs)
-    pkging, has_ver = optional_import("pkg_resources", name="packaging")
+    pkging, has_ver = optional_import("packaging", name="packaging")
     if has_ver:
         try:
-            return cast(bool, pkging.version.Version(lhs) <= pkging.version.Version(rhs))
-        except pkging.version.InvalidVersion:
+            return cast(bool, version(lhs) <= version(rhs))
```

Comment:
After the change above, you can just revert the change here.

Reply:
Hello,
Can you please clarify what you mean by 'revert the change here'? We took out references to pkging and changed the code in line 573 as requested.

Reply:
After change what I recommand above, the change here can just be reverted. Since you can use `pkging.version.Version(lhs)` to get the version from the `lhs`.
```
>>> pkging, has_ver = optional_import("packaging")
>>> pkging.version.Version("1.3.1a")
<Version('1.3.1a0')>
```